Eniko/index.js

```diff
@@ -0,0 +1,168 @@
+// 1. Check if a number is within a given range. Write a program that checks if a number is within the range of an object's min and max properties. Assume min <= max is always true.
+const numbers = {
+    min: 2,
+    max: 7,
+    getRange: function (num) {
+        if (num > this.min && num < this.max) {
```

Comment:
Task 1==>   num should be also equal this.min and max , if you try to pass 2 in console.log you will get false, but 2 is in range of 2 and 7